tests/Makefile.am

```diff
@@ -33,7 +34,8 @@ src_liblogger_la_CFLAGS = $(BUILD_FLAGS)
 # TESTS
 ################################################
 TESTING_FLAGS = -I system_config -I ../src -I include \
-                $(WARNING_FLAGS) $(CPPUNIT_CFLAGS)
+                -std=c++11 \
```

Comment:
Do we have to do this, for some of us still working on some rather old machines (but still in LTS)?